src/home-components/ArticleCardContainer.jsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+import { ArticleCard } from "./ArticleCard";
+export function ArticleCardContainer({ articleTopic }) {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    getArticles()
+      .then((articles) => {
+        setArticles(articles);
+        console.log(articles);
+      })
+      .catch((err) => console.log(err));
+  }, [getArticles]);
```

Comment:
the dependancy array watches values to see if they changed and get articles is a function and won't change value this should be left blank in this case.